MediaSession.cpp

```diff
@@ -307,6 +307,8 @@ MediaKeySession::~MediaKeySession(void) {
   if (DRM_REVOCATION_IsRevocationSupported())
     SAFE_OEM_FREE(m_pbRevocationBuffer);
 
+  Drm_Uninitialize( m_poAppContext );
```

Comment:
This looks wrong.
1. First of all it introduces a potential crash because MediaKeySession::UninitializeContext() below calls that and frees the context so you may be calling Drm_Uninitialize() on dangling pointer.
2. Having  said that my question would also be why MediaKeySession::UninitalizeContext() isn't called (it's called by PlayReady::DestroyMediaKeySession() which is called by AccessorOCDM::Remove() which is called from ~SessionImplementation() which should be called when Release() called on the other side makes the ref count to drop to 0).

Reply:
Moreover we need to apply same to OCDM-Playready-Nexus-SVP as it has same issue.

Reply:
I remember seeing this line and wondering about this myself as well (that's why I added the TODO (lazy way of fixing)). I think @pawel-stanek-ml is correct. I will move this code to the destructor and run some tests. And since this method only does this we can leave it empty. Removing might be a bit too soon if it is defined by the implemented interface, let's first see if other DRMs need it.

Reply:
Agreed. Could we also get rid of DestroyMediaKeySessionExt (it seems the only difference is calling this UninitializeContext() so once this is made redundant these also could be merged)? It seems we have this bug because of them. And again. Let's sync at least Playready-Nexus-SVP too

Reply:
See:
https://github.com/WebPlatformForEmbedded/ThunderNanoServices/commit/87d4d4bdec77ace8aa86ae2b2439a6a0c8d0f55b
https://github.com/WebPlatformForEmbedded/OCDM-Playready/commit/959897d16695545ae88d7077f92b2cd4cb2669b2
https://github.com/WebPlatformForEmbedded/Thunder/commit/38d616ed8a75f0b150049b401e71f2100832d7cb